src/listing.rs

```diff
@@ -275,13 +276,29 @@ pub fn directory_listing(
                     ));
                 } else if metadata.is_file() {
                     entries.push(Entry::new(
-                        file_name,
+                        file_name.clone(),
                         EntryType::File,
                         file_url,
                         Some(ByteSize::b(metadata.len())),
                         last_modification_date,
                         symlink_dest,
                     ));
+                    // TODO: Pattern match?
```

Comment:
I was thinking of supporting `readme.txt` as well, and `readme.org` like in github. But ended up not doing it. `txt`  should be easy enough to give it a try not sure about `org` file. I've removed the TODO for now. It can be separate issue later.